test

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/ruby --disable-gems
+
+# We're being very lazy here and assuming macOS with some preconditions:
```

Comment:
I would say it's worth setting a basic rspec / minitest / mocha and hooking up travis for linux and macos testing: https://docs.travis-ci.com/user/reference/osx/

Reply:
Yeah, I will probably do that next time I have to interact with this. I ran out of quality-improvement-motivation for the afternoon.